plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional, Union
+
+from hydra.core.config_store import ConfigStore
+
+
+@dataclass
+class DistributionConfig:
+    type: str
+    choices: Optional[List[Union[str, int, float]]] = None
+    low: Optional[float] = None
+    high: Optional[float] = None
+    log: bool = False
+    step: float = 1
+
+
+@dataclass
+class OptunaConfig:
+
+    # Direction of optimization
+    # "minimize" or "maximize"
+    direction: str = "minimize"
+
+    # Storage URL to persist optimization results
+    storage: Optional[str] = None
```

Comment:
Can we add some links here so that the user knows what are the valid values for each field? For example, https://optuna.readthedocs.io/en/stable/reference/storages.html?highlight=storage

Reply:
That makes sense. I added the link with an example storage URL in https://github.com/facebookresearch/hydra/pull/1132/commits/c3854e606d7abea22dcb6dd4fbca164aa43ea59c.